.github/workflows/pylint.yml

```diff
@@ -0,0 +1,38 @@
+name: Pylint
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    services:
+      mongodb:
+          image: mongo
+          ports:
+            - 27017:27017
+    strategy:
+      matrix:
```

Comment:
We do not really need a matrix. It would be enough to test on python 3.10 whether app passes lint and tests.